### ISSUE TITLE
IMAP Push Support

### ISSUE THREAD

**Issue by jojobear99** [jojobear99/PopTrayU#5]
Add IMAP Push notification support for instant updates when new messages update.

**Comment by jojobear99** [comment]
Currently this is blocked by the following indy bug: http://indy.codeplex.com/workitem/19628
This is a low priority issue for indy (over 3 years old) so it may require looking into alternate IMAP library options
